src/GovUk.Education.ExploreEducationStatistics.Publisher/Services/FileStorageService.cs

```diff
@@ -92,6 +93,15 @@ public async Task DeleteAllContentAsyncExcludingStaging()
             await DeleteBlobsAsync(publicContainer, string.Empty, excludePattern);
         }
 
+        public async Task DeletePreviousVersionContent(string publicationSlug, string previousVersionSlug)
+        {
+            // Delete previous content
```

Comment:
I make a change in #1844 to FileStorageService in an attempt to keep the FileStorageService more generic and keep the logic to file related tasks like add/move/copy/delete. I might update #1844 and move this to the ReleaseService to then call DeleteBlobAsync, if that's ok?